### PR TITLE
Dissolution forwardings with multiple consequent requests

### PR DESCRIPTION
**Pull request by openBackhaul**
Hi Vanitha,  

I wasn't sure about your proposed solution.  
Are your proposing 
- to change ForwardingList and CONFIGfile 
or
- to change ForwardingList, CONFIGfile and OAS, too?

This draft pull-request represents my request for you to assess, whether the proposed solution is satisfying your needs.
Please, comment and we will close the pull-request and apply your request on all forwardings.

Best regards,
               Thorsten

### CODE REVIEW

**Comment by vanithavalluripalli9** [comment]
Hi Thorsten , Thank you for the changes .

Your understanding is correct . The changes we requested are in config file and forwardings.yaml file only.

The changes also looks good . 

Only one small typo mistake , local-id of the forwardings 

- RequestForProvidingAcceptanceDataCausesDeterminingTheLanPortRole.VlanInterfaceKind
- RequestForProvidingAcceptanceDataCausesDeterminingTheLanPortRole.EthernetContainerStatus

should be "200" in config.json file.

File: spec/AutomatedLinkAcceptanceProxy+config.json [hunk 1]
```diff
@@ -3081,11 +3081,51 @@
...
+              }
+            ],
+            "fc-port": [
+              {
+                "local-id": "100",
+                "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_INPUT",
+                "logical-termination-point": "alap-1-0-0-op-s-is-000"
               },
               {
                 "local-id": "201",
```
Comment (vanithavalluripalli9) [inline spec/AutomatedLinkAcceptanceProxy+config.json:3106]: Can we change the local-id to "200".

File: spec/AutomatedLinkAcceptanceProxy+config.json [hunk 2]
```diff
@@ -3081,11 +3081,51 @@
...
+              }
+            ],
+            "fc-port": [
+              {
+                "local-id": "100",
+                "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_INPUT",
+                "logical-termination-point": "alap-1-0-0-op-s-is-000"
               },
               {
                 "local-id": "202",
```
Comment (vanithavalluripalli9) [inline spec/AutomatedLinkAcceptanceProxy+config.json:3131]: can we change local-id to "200"

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]

File: spec/AutomatedLinkAcceptanceProxy+config.json (hunk 2 above)
Comment (anacabbar) [reply spec/AutomatedLinkAcceptanceProxy+config.json:3131]: The local-id has been changed to "200"

File: spec/AutomatedLinkAcceptanceProxy+config.json (hunk 1 above)
Comment (anacabbar) [reply spec/AutomatedLinkAcceptanceProxy+config.json:3106]: The local-id has been changed to "200"

**Comment by anacabbar** [comment]
The localIds have been corrected and the other Forwardings have been dissolved as proposed

**Review by vanithavalluripalli9 — APPROVED** [review]
Hi Ana, Thank you for the changes .

Changes looks good. Approved the pull request.

**Pull request merged**